lib/pages/chat/events/message_download_content.dart

```diff
@@ -54,7 +60,7 @@ class MessageDownloadContent extends StatelessWidget {
             child: Row(
               children: [
                 Text(
-                  filetype,
+                  filetype ?? 'UNKNOWN',
```

Comment:
why is `UNKNOW`? how about other value?

Reply:
IMO, we have the default `mimeType`, not a `UNKNOWN`, how about `application/octet-stream`?